### DISCUSSION TITLE
Next.js를 어떻게 uiconverter에 녹여낼 것인가.

### DISCUSSION THREAD

**Discussion by chaaerim** [UIconverter/uiconverter#8] [Ideas]
- a -> Link 태그로 변경
- img -> Image 태그로 변경

이외에 더 고려할 수 있는 요소가 있는지 고민할 필요 있음.

**Comment by sonsurim** [comment]
img에 width, height가 없는 경우, Image 컴포넌트에는 어떻게 전달하면 좋을까요!? 👀

**Reply by chaaerim** [reply]
확인이 늦었군요 허헛 .. 둘다 없는 경우를 말씀하시는 건가요?! 아니면 비율 때문에 둘 중 하나만 없는 경우를 말씀하시는건가요?!
